src/pages/ca.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable max-len */
+import React from 'react';
+import CAProgram from '../components/CAProgram/CAProgram';
+import { PrivateRoute } from '../components';
+
+const CA = () => (
+  <PrivateRoute>
```

Comment:
I don't think privateRoute is required here as logon is not necessary to access the page.

Reply:
It is mentioned as part of CA Registration process, that one needs to be logged in before CA Registration